File: sdks/php/src/Render/Types/GenerateDocumentPreviewRequestType.php
<?php

namespace Pogodoc\Render\Types;

enum GenerateDocumentPreviewRequestType: string
{
    case Docx = "docx";
    case Xlsx = "xlsx";
    case Pptx = "pptx";
    case Ejs = "ejs";
    case Html = "html";
    case Latex = "latex";
    case React = "react";
}

File: sdks/csharp/sdk/src/PogodocApi/Documents/Types/GenerateDocumentPreviewRequestType.cs
using System.Runtime.Serialization;
using System.Text.Json.Serialization;
using PogodocApi.Core;

namespace PogodocApi;

[JsonConverter(typeof(EnumSerializer<GenerateDocumentPreviewRequestType>))]
public enum GenerateDocumentPreviewRequestType
{
    [EnumMember(Value = "docx")]
    Docx,

    [EnumMember(Value = "xlsx")]
    Xlsx,

    [EnumMember(Value = "pptx")]
    Pptx,

    [EnumMember(Value = "ejs")]
    Ejs,

    [EnumMember(Value = "html")]
    Html,

    [EnumMember(Value = "latex")]
    Latex,

    [EnumMember(Value = "react")]
    React,
}
